compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BStreamType.java

```diff
@@ -37,7 +37,7 @@
     public BStreamType(int tag, BType constraint, BType error, BTypeSymbol tsymbol) {
         super(tag, tsymbol);
         this.constraint = constraint;
-        this.error = error;
+        this.error = error != null ? error : new BNeverType();
```

Comment:
Cant we use `symTable.neverType` here?

Reply:
We can. But, is it ok to import symTable to a type?